types/pg-query-stream/index.d.ts

```diff
@@ -19,6 +19,7 @@ declare class QueryStream extends stream.Readable {
     text: string;
     values?: any[];
     constructor(text: string, values?: any[], options?: QueryStream.Options);
+    submit(connection: any): void;
```

Comment:
I do not think you want to use any here. It seems to be an interface of a submit function right?

Reply:
The parameter is a pg connection, but there is no abstraction for the Connection class in the pg types, that's why I chose `any` type. Should I make an abstraction for the connection class?

Reply:
That would be preferable. An `any` is usually a hole for bugs. If you really don't want to specify the type, `{}` is preferable since the receiver cannot assume anything about the argument.

Reply:
I've added the Connection class, can you review it again?